.ci/docker/common/install_mkl.sh

```diff
@@ -4,12 +4,13 @@ set -ex
 # MKL
 MKL_VERSION=2024.2.0
 
-mkdir -p /opt/intel/
+MKLROOT=${MKLROOT:-/opt/intel}
```

Comment:
@atalman IIUC, this script *could* end up using a different value of $MKLROOT (if env var already exists) as compared to `.ci/docker/common/install_rocm_magma.sh`, which will get it as `/opt/intel` from the ENV directive [here](https://github.com/pytorch/pytorch/pull/129022/commits/e2a145363268326e20d3e245bdfd407b67ce2f83#diff-2cbe032767323860818626bee279d378c94a96a8f3481495ea3e37af8ee53116R185). If that happens, the ROCm magma build would fail. Either we should hardcode `MKLROOT` to `/opt/intel` here, or set `ARG MKLROOT /opt/intel` [as in my PR](https://github.com/pytorch/builder/pull/1919/files#diff-daadaa101b2e2fec84e157f4a4e80eb8feab78a0ee7a8cd9c3a99348b54f4d03R6), since ARGs are visible in different stages of a multi-stage Dockerfile, but ENVs are limited to the specific stage where they are specified?

Reply:
Sounds good let me use '/opt/intel' here . I don't think you are calling this file from rocm builds as far as I know.